### DISCUSSION TITLE
[Feature Request] Pump Holds / Hold Combo Ticks in all modes

### DISCUSSION THREAD

**Discussion by aelteria** [TeamRizu/OutFox#253] [03 - Feature Requests]
Combo Ticks should be able to work on all modes, including outside of Pump.
From what I know, it's hard-coded in **GameManager.cpp** to only work in this specific mode.

As for misses, they shouldn't drain health too hard, as it would miss per tick(?), and would drain health too fast.

**Comment by Scraticus** [comment]
Hiya Aelteria!

We have edited how holds work in para mode, but how would this feature be used?

As far as i'm aware, i'm not sure of any use in other modes, would it be a mod/simfile call or something else?

Thanks for the suggestion, but let us know what could/would be done with it.

Thanks

**Comment by aelteria** [comment]
For instance in ArrowVortex, the SM5 Tempo menu actually allows the use of combo ticks, but never works in-game outside of the 5-panel / Pump mode. As for what could be done with it, it'd be useful in attack or gimmick files, or just for visuals. It'd be interesting to see what people do with it, and could partner well with combo hit multipliers maybe?
It actually shows up in the .ssc when you save it in .ssc format, but never gets used.
It feels more like a bug than anything else.
